sherpa/models/tests/test_model.py

```diff
@@ -868,6 +869,24 @@ def check_cache(mdl, expected, x, xhi=None):
     assert cache[digest] == pytest.approx(expected)
 
 
+class ExpcacheTestModel(RegriddableModel1D):
+    '''A model to test the modelCacher1d_exp decorator.
+
+    Note that a direct calc  gives different values than when
+    using the cache. Obviously, that's not what a real model
+    would do, but it's useful for testing.
```

Comment:
Is the point here that `calc` doesn't scale the `a(x)` term - here `np.ones_like` - by an exponential?